Qast/ViewControllers/SoundZoneDetailViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SoundZoneDetailViewController.swift
+//  Qast
+//
+//  Created by Andrew O'Brien on 7/22/19.
+//  Copyright © 2019 Qast. All rights reserved.
+//
+
+import UIKit
+
+class SoundZoneDetailViewController: UIViewController {
+
+    let soundZone: SoundZone?
+    let soundZoneTitle = UILabel()
+    let dismissIcon = UIImageView(image: UIImage(named: "x-button"))
+    
+    init(_ soundZone: SoundZone) {
+        self.soundZone = soundZone
+        super.init(nibName: nil, bundle: nil)
+        view.backgroundColor = .white
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupText()
+        setupDismissIcon()
+    }
+
+}
+
+// MARK: UI Setup
+
+extension SoundZoneDetailViewController {
+    func setupDismissIcon() {
+        view.addSubview(dismissIcon)
+        dismissIcon.snp.makeConstraints { (make) in
+            make.width.equalTo(40)
```

Comment:
You can chain these together if you would like. It would look something like `make.height.width.equal(40)`

Reply:
nice